src/extension/wps/src/main/java/au/org/emii/wps/GoGoDuckProcess.java

```diff
@@ -1,47 +1,49 @@
 package au.org.emii.wps;
 
 import java.io.File;
+import java.nio.file.DirectoryStream.Filter;
+import java.util.HashMap;
 import java.util.Map;
 
-import au.org.emii.gogoduck.worker.GoGoDuckException;
-import au.org.emii.gogoduck.worker.GoGoDuck;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.xpath.XPath;
+import javax.xml.xpath.XPathConstants;
+import javax.xml.xpath.XPathExpression;
+import javax.xml.xpath.XPathFactory;
 
-import au.org.emii.gogoduck.worker.URLMangler;
 import net.opengis.wps10.ExecuteType;
+
 import org.apache.commons.io.FilenameUtils;
 import org.geoserver.config.impl.GeoServerImpl;
 import org.geoserver.ows.Dispatcher;
 import org.geoserver.platform.GeoServerResourceLoader;
 import org.geoserver.platform.Operation;
-import org.geoserver.platform.exception.GeoServerRuntimException;
-import org.geoserver.wps.WPSException;
-import org.geoserver.wps.WPSInfo;
-import org.opengis.util.ProgressListener;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
+import org.geoserver.wps.gs.GeoServerProcess;
+import org.geoserver.wps.process.FileRawData;
+import org.geoserver.wps.process.RawData;
+import org.geoserver.wps.process.StringRawData;
+import org.geoserver.wps.resource.WPSResourceManager;
 import org.geotools.process.ProcessException;
 import org.geotools.process.factory.DescribeParameter;
 import org.geotools.process.factory.DescribeProcess;
 import org.geotools.process.factory.DescribeResult;
-import org.geoserver.wps.gs.GeoServerProcess;
-import org.geoserver.wps.process.FileRawData;
-import org.geoserver.wps.resource.WPSResourceManager;
+import org.geotools.process.factory.DescribeResults;
+import org.opengis.util.ProgressListener;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.w3c.dom.Document;
 
-import javax.xml.parsers.DocumentBuilder;
-import javax.xml.parsers.DocumentBuilderFactory;
-import javax.xml.xpath.XPath;
-import javax.xml.xpath.XPathConstants;
-import javax.xml.xpath.XPathExpression;
-import javax.xml.xpath.XPathFactory;
+import au.org.emii.gogoduck.worker.GoGoDuck;
+import au.org.emii.gogoduck.worker.URLMangler;
 
 @DescribeProcess(title="GoGoDuck", description="Subset and download gridded collection as NetCDF files")
-public class GoGoDuckProcess implements GeoServerProcess {
+public class GoGoDuckProcess implements GeoServerProcess, SubsetOperation {
+    private static final String RESULT_TYPE = "application/x-netcdf";
+
     private static final Logger logger = LoggerFactory.getLogger(GoGoDuck.class);
```

Comment:
I know you didn't write this but I see that this logger is also created for the wrong class.

Could we fix this one too, please.